css/css-values/acos-asin-atan-atan2-serialize.html

```diff
@@ -43,9 +43,6 @@
     "atan(0.5)"                                         :"calc(26.5651deg)",
     "atan(0.577350269)"                                 :"calc(30deg)",
     "atan(0)"                                           :"calc(0deg)",
-    "atan(infinity)"                                    :"calc(90deg)",
```

Comment:
We didn't undefine this; it still needs to return 90deg.

Reply:
Good point, fixed, thanks!